Formula/qhull.rb

```diff
@@ -0,0 +1,19 @@
+class Qhull < Formula
+  desc "Code for computing convex hulls in n dimensions"
+  homepage "http://www.qhull.org/"
+  url "http://www.qhull.org/download/qhull-2015-src-7.2.0.tgz"
+  version "2015.2"
+  sha256 "78b010925c3b577adc3d58278787d7df08f7c8fb02c3490e375eab91bb58a436"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    result = shell_output("#{bin}/rbox c D2 | #{bin}/qconvex s n 2>&1")
```

Comment:
We could probably convert everything to just use shell_output, but `pipe_output` is intended for this case.
```
pipe_output("#{bin}/qconvex s n 2>&1", "#{bin}/rbox c D2", 0)
```